apps/web/src/lib/studies.ts

```diff
@@ -266,10 +266,30 @@ export const getStudiesForExport = async (organisationIds: number[]) => {
 
 export type UpdateStudyInput = Prisma.StudyUpdateInput
 
+export const mapCPMSStatusToSEStatus = (cpmsStatus: string): string => {
+  const statusMap: { [key: string]: string } = {
```

Comment:
I know there has been some discussion around this, but using a hardcoded mapping of these statuses using the string value seems like a bad idea.

These strings could change in either system at any time, especially if they are used for display purposes.

Having the mapping in the database would be 'less bad'.